MetricsReloaded/utility/assignment_localization.py

```
@@ -554,7 +637,7 @@ def resolve_ambiguities_matching(self):
                 list_matching = []
                 for (r, c) in zip(row, col):
```

Ruff (0.15.10)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)